-------------------------------------------------------------------------------
Test set: com.msg.immun.ImmunApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.095 s <<< FAILURE! - in com.msg.immun.ImmunApplicationTests
contextLoads(com.msg.immun.ImmunApplicationTests)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.msg.immun.model.VerificationToken.patientModel references an unknown entity: com.msg.immun.model.PatientModel
Caused by: org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.msg.immun.model.VerificationToken.patientModel references an unknown entity: com.msg.immun.model.PatientModel

